src/ServiceInsight/Saga/SagaResources.xaml

```diff
@@ -25,4 +25,13 @@
     
     <SolidColorBrush x:Key="FailedMessageBackground" Color="#F9916F" />
     <SolidColorBrush x:Key="RetriedMessageBackground" Color="#F2F2F2" />
+
+    <Style x:Key="GrayNullTextStyle" TargetType="{x:Type TextBlock}">
```

Comment:
If it changes to bold, should we question it then and rename accordingly? I'm open to other names but this is not a generic text style so don't think `SagaValueTextStyle` would be a good choice.